Makefile

```diff
@@ -563,6 +565,7 @@ CC		:= $(CONFIG_CROSS_COMPILE)$(CONFIG_COMPILER)
 CPP		:= $(CC)
 CXX		:= $(CPP)
 GOC		:= $(CONFIG_CROSS_COMPILE)gccgo-7
+RUSTC		:= rustc
```

Comment:
How would cross compiling work with rust? I would have expected something like
```Makefile
RUSTC		:= $(CONFIG_CROSS_COMPILE)rustc
```

Reply:
Rustc is using --target for specifying the target, I've modified the code accordingly.

Reply:
I think `$(CONFIG_CROSS_COMPILE)` includes a minus. We probably just need to remove the trailing `-` from the variable. However, do you know if the naming is exactly the same or do we need translating it? A tiny support script could do the job, e.g., `support/scripts/rustc-cross`